SMLHelper/Patchers/WaterBiomeManagerPatcher.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using HarmonyLib;
+using System.Reflection.Emit;
+namespace SMLHelper.V2.Patchers
+{
+    internal class WaterBiomeManagerPatcher
+    {
+            [HarmonyPatch(typeof(WaterBiomeManager), nameof(WaterBiomeManager.Start))]
+            [PatchUtils.Prefix]
+            internal static bool WaterBiomeManager_Start_Prefix(WaterBiomeManager __instance)
+            {
+                for (var i = 0; i < BiomeThings.Variables.Biomes.Count; i++)
+                {
+                    var biome = BiomeThings.Variables.Biomes[i];
+                    var settings = new WaterBiomeManager.BiomeSettings()
+                    {
+                        name = biome.BiomeName,
+                        settings = biome.WaterScapeSettings,
+                        skyPrefab = new UnityEngine.GameObject("Useless Junk.")
+                    };
+                    __instance.biomeSettings.Add(settings);
+                }
+                return true;
+            }
+            [HarmonyPatch(typeof(WaterBiomeManager), nameof(WaterBiomeManager.Start))]
```

Comment:
Remove this patch - `AtmosphereVolume` already covers it.

Reply:
so is everything in this file redundant?